QModManager/QMod.cs

```diff
@@ -40,15 +40,38 @@ internal static QMod FromJsonFile(string file)
                 string json = File.ReadAllText(file);
                 QMod mod = JsonConvert.DeserializeObject<QMod>(json);
 
+                if (mod == null) return null;
+
+                if (mod.ForBelowZero == true) mod.Game = Patcher.Game.BelowZero;
+                else mod.Game = Patcher.Game.Subnautica;
+
                 return mod;
             }
             catch (Exception e)
             {
-                Console.WriteLine("ERROR! mod.json deserialization failed!");
-                Console.WriteLine(e.ToString());
+                Logger.Error($"\"mod.json\" deserialization failed for file \"{file}\"!");
+                Debug.LogException(e);
 
                 return null;
             }
         }
     }
+    
+    internal class FakeQMod : QMod
```

Comment:
This can be replaced by a static method that returns a normal QMod but with the updated values.

Reply:
Addressed in commit 60f9a4b